canonical/canonical.com
# Fix landing form submission

## Done

- Fixed form action url and hidden fields  

## QA

- View the site locally in your web browser at: https://canonical-com-1276.demos.haus/
- Click on "Contact us" hero CTA and fill out the form
- See that it submits correctly 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12323
